zerver/tests/tests.py

```diff
@@ -2330,9 +2330,12 @@ def _test_cases(self, tokens, msg_id, body, send_as_user):
         othello = get_user_profile_by_email('othello@zulip.com')
         hamlet = get_user_profile_by_email('hamlet@zulip.com')
         handle_missedmessage_emails(hamlet.id, [{'message_id': msg_id}])
-
+        reply_to_addresses = "noreply@example.com"
         msg = mail.outbox[0]
-        reply_to_addresses = [settings.EMAIL_GATEWAY_PATTERN % (u'mm' + t) for t in tokens]
```

Comment:
When trying to test the case of `EMAIL_GATEWAY_PATTERN=''`, this line threw `TypeError` exception and thus, didn't let the test run. I was thinking writing a separate function like `_test_text_in_email_body` so that the settings of other email headers don't need to be a concern while testing templates.